prepare_tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Can you please add Apache 2.0 license headers to those non-code related files and move them to `contrib/`?